service/project.go

```diff
@@ -586,6 +601,14 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 	for i, buildDef := range responseRef.PeriodicBuilds {
 		catcher.Wrapf(buildDef.Validate(), "invalid periodic build definition on line %d", i+1)
 	}
+	for _, containerResource := range responseRef.ContainerSizes {
+		if *containerResource.CPU <= 0 {
```

Comment:
I would use `utility.ToIntPtr(containerResource.CPU) < 0` since the goroutine will panic if CPU is nil.